src/algokit_utils/_transfer.py

```diff
@@ -37,7 +35,27 @@ class TransferParameters:
     if this is set it's possible the transaction could get rejected during network congestion"""
 
 
-def _check_fee(transaction: PaymentTxn, max_fee: int | None) -> None:
+@dataclasses.dataclass(kw_only=True)
+class TransferParameters(TransferParametersBase):
+    """Parameters for transferring µALGOs between accounts"""
+
+    micro_algos: int
+
+
+@dataclasses.dataclass(kw_only=True)
+class TransferAssetParameters(TransferParametersBase):
```

Comment:
Please use docstrings instead of raw comments per each parameter. WIll be better for intellisence too

Something like this for example:

```
@dataclasses.dataclass(kw_only=True)
class TransferAssetParameters(TransferParametersBase):
    """Parameters for transferring assets between accounts.

    Args:
        asset_id (int): The asset ID to be transferred.
        amount (int): The amount of the asset to be sent in the transfer.
        clawback_from (Account | str | None: An address of a target account from which to perform a
            clawback operation. If provided, it must be an instance of the Account class, a string representing an
            account address, or None. Defaults to None.
    """

    asset_id: int
    amount: int
    clawback_from: Account | str | None = None
```